src/Illuminate/Testing/TestResponse.php

```diff
@@ -170,7 +170,24 @@ public function assertStatus($status)
      */
     protected function statusMessageWithDetails($expected, $actual)
     {
-        return "Expected response status code [{$expected}] but received {$actual}.";
+        $message = "Expected response status code [{$expected}] but received {$actual}.";
+
+        if ($actual === 500 && $lastException = $this->exceptions->last()) {
+            $message .= <<<"EOF"
+
+
+            The following exception occurred during the last request:
+
+            $lastException
+
+            ----------------------------------------------------------------------------------
+
+            {$lastException->getMessage()}
+
+            EOF;
```

Comment:
```suggestion
            $exception = ThrowableBuilder::from($lastException);

            $message .= <<<"EOF"

            The following exception occurred during the last request:

            {$exception->asString()}

            ----------------------------------------------------------------------------------

            {$lastException->getMessage()}
            EOF;
```

### Before

![CleanShot 2024-06-05 at 23 41 47](https://github.com/laravel/framework/assets/172966/4484d144-89af-443c-84af-30125ff8c174)
![CleanShot 2024-06-05 at 23 41 56](https://github.com/laravel/framework/assets/172966/590bc1d8-bb3d-417b-823c-e5677d01fdd8)

#### After 

![CleanShot 2024-06-05 at 23 41 16](https://github.com/laravel/framework/assets/172966/e774e67f-3672-47b7-81d4-8d56315269c3)

Reply:
Also need to add the following import:

```php
use PHPUnit\Event\Code\ThrowableBuilder;
```